### ISSUE TITLE
Clickhouse is a popular analytics data warehouse (e.g. Plausible); support it's SQL-like queries

### ISSUE THREAD

**Issue by blaggacao** [frappe/insights#271]
https://clickhouse.com/

https://clickhouse-driver.readthedocs.io/en/latest/

Example query (Plausible Schema):
```sql
SELECT *
FROM events_v2
WHERE lower(substring(utm_source, 1, 4)) = 'cust'
```

**Comment by blaggacao** [comment]
Will be closed by: https://github.com/frappe/insights/pull/270

**Comment by weeebdev** [comment]
> Will be closed by: [#270](https://github.com/frappe/insights/pull/270)

May I ask, how is it implemented? Do I have to use DuckDB to connect clickhouse?
